README.md

```diff
@@ -1319,180 +1319,72 @@ err := descopeClient.Management.Audit().CreateEvent(context.Background(), &desco
-                      expression:
-                          neType: relationRight
-                          relationDefinition: parent
-                          relationDefinitionNamespace: doc
-                          targetRelationDefinition: viewer
-                          targetRelationDefinitionNamespace: folder
-                    - nType: child
-                      expression:
-                          neType: targetSet
-                          targetRelationDefinition: editor
-                          targetRelationDefinitionNamespace: doc
 ```
+model AuthZ 1.0
+
+type user
+
+type org
+  relation member: user
+  relation parent: org
+
+type folder
+  relation parent: folder
+  relation owner: user | org#member
+  relation editor: user 
+  relation viewer: user 
+
+  permission can_create: owner | parent.owner
+  permission can_edit: editor | can_create
+  permission can_view: viewer | can_edit
+
+type doc
+  relation parent: folder
+  relation owner: user | | org#member
+  relation editor: user 
+  relation viewer: user 
+
+  permission can_create: owner | parent.owner
+  permission can_edit: editor | can_create
+  permission can_view: viewer | can_edit
+
 
 Descope SDK allows you to fully manage the schema and relations as well as perform simple (and not so simple) checks regarding the existence of relations.
 
 ```go
-// Load the existing schema
-schema, err := descopeClient.Management.Authz().LoadSchema(context.Background())
-if err != nil {
-    // handle error
-}
-
-// Save schema and make sure to remove all namespaces not listed
-err := descopeClient.Management.Authz().SaveSchema(context.Background(), schema, true)
+// Save schema
+err := descopeClient.Management.FGA().SaveSchema(context.Background(), schema)
 
 // Create a relation between a resource and user
-err := descopeClient.Management.Authz().CreateRelations(context.Background(), []*descope.AuthzRelation {
-    {
-        resource: "some-doc",
-        relationDefinition: "owner",
-        namespace: "doc",
-        target: "u1",
+err := descopeClient.Management.FGA().CreateRelations(context.Background(), []*descope.FGARelation {
+    {   
+		Resource: "some-doc", 
+		ResourceType: "doc", 
+		Relation: "owner", 
+		Target: "u1", 
+		TargetType: "user"
     },
 })
 
-// Check if target has the relevant relation
-// The answer should be true because an owner is also a viewer
-relations, err := descopeClient.Management.Authz().HasRelations(context.Background(), []*descope.AuthzRelationQuery{
+// Check if target has a relevant relation
+// The answer should be true because an owner can also view
+relations, err := descopeClient.Management.FGA().Check(context.Background(), []*descope.FGARelation{
     {
-        resource: "some-doc",
-        relationDefinition: "viewer",
-        namespace: "doc",
-        target: "u1",
+		Resource: "some-doc", 
+		ResourceType: "doc", 
+		Relation: "can_view", 
+		Target: "u1", 
+		TargetType: "user"
     }
-})
-```
+}) -->
```

Comment:
Extraneous arrow?

Reply:
actually seems the arrow needs to be closing code syntax ```

Reply:
<img width="1133" alt="Screenshot 2024-11-10 at 2 56 27 PM" src="https://github.com/user-attachments/assets/6fd6292a-23a0-495f-a48b-cb7b41062fdd">